<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string extracomment="选定网元"/>
  </property>
  <property name="accessibleDescription">
   <string extracomment="选定网元"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>721</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCell">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>260</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNode">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="viewKPI">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>选网元、属性和日期
点击显示折线图</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxKPI">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string>选定网元</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2016</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-M-d</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>280</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2018</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d</string>
    </property>
   </widget>
   <widget class="QLabel" name="datelabel1">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>起始时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="datelabel2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>结束时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="B_cell">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>200</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查看小区信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_node">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>330</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查看基站信息</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxKPI_p">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string>选定网元</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>721</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="B_save">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>490</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>保存当前表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_gen">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>生成表格</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genPara">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>190</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="genTips">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>请输入参数
以生成表格</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="B_gen_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>显示表格</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="date">
     <date>
      <year>2016</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>350</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>10</hour>
      <minute>40</minute>
      <second>0</second>
      <year>2016</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="date">
     <date>
      <year>2016</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_PRB_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>420</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>小时一致按分钟粒度查询tbPRBnew
否则按小时粒度查询tbPRB</string>
    </property>
   </widget>
   <widget class="QLabel" name="PRBlabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>260</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>调整时间 hh:mm:ss
开始范围和结束范围</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxKPI_DAY">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string>选定网元</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxKPI_p_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string>选定网元</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxKPI_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string>选定网元</string>
    </property>
   </widget>
   <zorder>comboBoxKPI_p</zorder>
   <zorder>tableView</zorder>
   <zorder>B_cell</zorder>
   <zorder>comboBoxNode</zorder>
   <zorder>B_node</zorder>
   <zorder>comboBoxCell</zorder>
   <zorder>viewKPI</zorder>
   <zorder>comboBoxKPI</zorder>
   <zorder>dateEdit_1</zorder>
   <zorder>dateEdit_2</zorder>
   <zorder>datelabel1</zorder>
   <zorder>datelabel2</zorder>
   <zorder>customPlot</zorder>
   <zorder>B_save</zorder>
   <zorder>B_gen</zorder>
   <zorder>genPara</zorder>
   <zorder>genTips</zorder>
   <zorder>B_gen_2</zorder>
   <zorder>dateTimeEdit_1</zorder>
   <zorder>dateTimeEdit_2</zorder>
   <zorder>B_PRB_1</zorder>
   <zorder>PRBlabel</zorder>
   <zorder>comboBoxKPI_DAY</zorder>
   <zorder>comboBoxKPI_p_2</zorder>
   <zorder>comboBoxKPI_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据查询</string>
    </property>
    <addaction name="actiontbCell"/>
    <addaction name="actiontbKPI"/>
    <addaction name="actiontbPRB"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>生成新表</string>
    </property>
    <addaction name="actiontbC2Inew"/>
    <addaction name="actiontbC2I3"/>
    <addaction name="actiontbPRBnew"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>数据导入</string>
    </property>
    <addaction name="actiontbCell_3"/>
    <addaction name="actiontbKPI_2"/>
    <addaction name="actiontbPRB_2"/>
    <addaction name="actiontbMROData_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiontbCell">
   <property name="text">
    <string>tbCell</string>
   </property>
  </action>
  <action name="actiontbMROData">
   <property name="text">
    <string>tbMROData</string>
   </property>
  </action>
  <action name="actiontbPRB">
   <property name="text">
    <string>tbPRB</string>
   </property>
  </action>
  <action name="actiontbKPI">
   <property name="text">
    <string>tbKPI</string>
   </property>
  </action>
  <action name="actiontbC2Inew">
   <property name="text">
    <string>tbC2Inew</string>
   </property>
  </action>
  <action name="actiontbC2I3">
   <property name="text">
    <string>tbC2I3</string>
   </property>
  </action>
  <action name="actiontbPRBnew">
   <property name="text">
    <string>tbPRBnew</string>
   </property>
  </action>
  <action name="actionlistUser">
   <property name="text">
    <string>listUser</string>
   </property>
  </action>
  <action name="actiontbCell_2">
   <property name="text">
    <string>tbCell</string>
   </property>
  </action>
  <action name="actiontbKPI_2">
   <property name="text">
    <string>tbKPI</string>
   </property>
  </action>
  <action name="actiontbPRB_2">
   <property name="text">
    <string>tbPRB</string>
   </property>
  </action>
  <action name="actiontbMROData_2">
   <property name="text">
    <string>tbMROData</string>
   </property>
  </action>
  <action name="actiontbCell_3">
   <property name="text">
    <string>tbCell</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
